raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -178,6 +179,14 @@ def is_send_transfer_almost_equal(
 
 
 def filter_available_routes(transfers_pair, routes):
+    """This function makes sure we don't try routes that have already been tried
```

Comment:
What about `Filter routes that have been used`, and maybe renaming the function to `filter_used_routes`?